taxi/admin.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
-from .models import Driver, Car, Manufacturer
+
+from .models import Car, Driver, Manufacturer
```

Comment:
Consider adding a comment or docstring to explain the purpose of importing `Car`, `Driver`, and `Manufacturer`. This can help other developers understand the context and usage of these models in the admin interface.